backend/backend.py

```diff
@@ -17,10 +17,11 @@ class Backend(ABC):
         Handle combining the video feed with overlays and displaying, and
         recording the video feed to a file. """
 
-    def __init__(self, width: int, height: int, publish_recording_status_func: PublishFunc, exception_handler: PublishFunc):
+    def __init__(self, width: int, height: int, publish_recording_status_func: PublishFunc, publish_camera_is_online_func: PublishFunc, exception_handler: PublishFunc):
```

Comment:
I think the length of the line is not really important at this point because if we want to reduce it, we should prevent the method from needing to be passed two functions in its parameters.

Reply:
Yeah thats fair enough